data.sql

```diff
@@ -31,4 +31,38 @@ UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
 UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu');
 UPDATE animals SET owner_id = 3 WHERE name IN ('Devimon', 'Plantmon');
 UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
-UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon');
\ No newline at end of file
+UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon');
+
+/* Many to many inputs */
+/* Vets input */
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('William Tatcher', 45, '2000-04-23');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Maisy Smith', 26, '2019-01-17');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Stephanie Mendez', 64, '1981-05-04');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Jack Harkness', 38, '2008-06-08');
+
+/* Specialties input */
+INSERT INTO specialties (species_id, vets_id) VALUES (1,1), (2,3), (1,3), (2,4);
+
+/* visits input */
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (3, 1, '2020-05-24');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (3, 3, '2020-07-22');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (4, 4, '2021-02-02');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (1, 2, '2020-01-05');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (1, 2, '2020-03-08');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (1, 2, '2020-05-14');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (6, 3, '2021-05-04');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (7, 4, '2021-02-24');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (9, 2, '2019-12-21');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (9, 1, '2020-08-10');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (9, 2, '2020-04-07');
```

Comment:
- [ ] Kindly change this date to ```2021-04-07``` as this would help you have more accurate data.